packages/grafana-data/src/datetime/formats.ts

```diff
@@ -98,11 +98,13 @@ export function localTimeFormat(
 
   // https://momentjs.com/docs/#/displaying/format/
   const parts = new Intl.DateTimeFormat(locale, options).formatToParts(new Date());
+  const hour12 = !!parts.filter((part) => part.type === 'dayPeriod').pop();
```

Comment:
Could we do
```suggestion
  const dateTimeFormat = new Intl.DateTimeFormat(locale, options);
  const parts = dateTimeFormat.formatToParts(new Date());
  const hour12 = dateTimeFormat.resolvedOptions().hour12;
```
insetad?